src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -153,7 +153,7 @@
                                                               :defaultValue description
                                                               :style {:width 400}
                                                               :rows 5}))
-                description description
+                description [:div {:style {:whiteSpace "pre"}} description]
```

Comment:
`:whiteSpace "pre"` has the same semantics as the `<pre>` tag. The problem with it is that it will also preserve the lack of line wraps, so if the user types an entire paragraph into the box, the whole thing will be forced into a single line.

`pre-wrap` preserves white space and line wraps while also adding soft wraps when the content reaches the edge of the container.